src/lib/probe.c

```diff
@@ -70,9 +70,10 @@ static char *cbm_get_luks_uuid(const char *part)
         const char *sys = cbm_system_get_sysfs_path();
 
         /* i.e. /sys/block/dm-1/slaves/dm-0/slaves/sdb1/dev */
-        npath = string_printf("%s/block/%s/slaves/*/slaves/*/dev", sys, part);
+        /* i.e. /sys/block/dm-1/slaves/sdb1/dev */
```

Comment:
This works for now but it will need to be altered this once we have more lvm support. The part being passed in may end up being stacked on another virtual device and so function will need to support recursing the device slaves. Really more just a note for the future =).